spec/data/flatten/src/test.cfhighlander.rb

```diff
@@ -0,0 +1,42 @@
+CfhighlanderTemplate do
+
+  # explicit configuration for vpc component
+  vpc_config = { 'maximum_availability_zones' => 5 }
+
+  hosted_zone = FnJoin('.',[Ref('AWS::Region'), Ref('AWS::AccountId'), 'cfhighlander.info'])
+  vpc_template = 'github:toshke/hl-component-vpc#inline_poc'
```

Comment:
maybe push these to a branch on the upstream repos and they update the test. We can then update the tests once they are merged. Or should be even reference external components in the core tests? maybe create a seperate testsuite?

Reply:
`test.cfhighlander.rb` is actually not used in rspec tests - `c.cfhighlander.rb` is. i've removed this file for commit - it was there as an example.